### PR TITLE
Fixup data visualization

### PR DESCRIPTION
**Pull request by Museau**
- [x] - Removed unused functions
- [x] - Fixup variable name ("identification" -> "anomaly_id")

### CODE REVIEW

File: src/enhanced-patchcore/notebooks/visualize_labels.ipynb [hunk 1]
```diff
@@ -408,7 +409,7 @@
    "source": [
     "# Visualize masks with anomaly on the border\n",
     "nb_border_pixel = 20  # To adapt: number of pixel to include in the border\n",
-    "for idx, row in df_anomalous.iterrows():\n",
+    "for idx, row in df_anomalous[df_anomalous[\"pass_id\"] == 1].iterrows():\n",
```
Comment (akshathaarodi) [inline src/enhanced-patchcore/notebooks/visualize_labels.ipynb:412]: why pass_id == 1?

Comment (Museau) [reply src/enhanced-patchcore/notebooks/visualize_labels.ipynb:412]: We have the masks only for the first video pass for each cable.

Comment (akshathaarodi) [reply src/enhanced-patchcore/notebooks/visualize_labels.ipynb:412]: I see. I asked because I thought we were showing bbox for all passes.

**Review by akshathaarodi — APPROVED** [review]

File: src/enhanced-patchcore/notebooks/visualize_labels.ipynb (hunk 1 above)
Comment (Museau) [reply src/enhanced-patchcore/notebooks/visualize_labels.ipynb:412]: We do have bounding boxes for all passes but here we are visualizing the masks.

Comment (akshathaarodi) [reply src/enhanced-patchcore/notebooks/visualize_labels.ipynb:412]: Got it!

**Pull request merged**